backup/prod/feeds.yaml

```diff
@@ -5314,128 +5314,6 @@ AMPL-USD-VWAP-1day:
       calculator:
         name: VWAP
 
-# GVol
```

Comment:
This file shouldn't be here. It's a backup file under /prod.

Reply:
I'll remove this file.

Reply:
This file is created when we run the command `npm run script:backupFeeds` not sure if it is right to remove the file.